install/gcp/installer/forseti_installer.py

```diff
@@ -307,10 +308,14 @@ def wait_until_vm_initialized(self, vm_name):
         loading_message = ('Waiting for Forseti {} to be initialized...'
                            .format(installation_type))
 
-        _ = utils.start_loading(
-            max_loading_time=constants.MAXIMUM_LOADING_TIME_IN_SECONDS,
-            exit_condition_checker=status_checker,
-            message=loading_message)
+        try:
+            _ = utils.start_loading(
+                max_loading_time=constants.MAXIMUM_LOADING_TIME_IN_SECONDS,
+                exit_condition_checker=status_checker,
+                message=loading_message)
+        except installer_errors.SSHError:
+            # Expected error, don't need to do anything.
```

Comment:
It is good to have a comment here.  But I would clarify why this is an expected error, and why we don't need to do anything.

Reply:
Updated.